f2/src/forms/MidFeedbackForm.js

```diff
@@ -1,60 +1,61 @@
 /** @jsx jsx */
 import React, { useState } from 'react'
+import PropTypes from 'prop-types'
 import { jsx } from '@emotion/core'
 import { H1, H2 } from '../components/header'
```

Comment:
Should use the <P> component from f2\src\components\formik\paragraph\index.js. You may have to style a new component for H1 and H2, let me know if you need help.